README.md

```diff
@@ -125,10 +169,30 @@ following `settings.json`:
 }
 ```
 
+And, for Jupyter Notebooks:
+
+```json
+{
+  "notebook.formatOnSave.enabled": true,
+  "notebook.codeActionsOnSave": {
+    "source.fixAll": true,
+    "source.organizeImports": true
+  },
+  "[python]": {
+    "editor.defaultFormatter": "charliermarsh.ruff"
+  }
+}
+```
+
 _Note: if you're using Ruff to organize imports in VS Code and _also_ expect to run Ruff from the
 command line, you'll want to enable Ruff's isort rules by by adding `"I"` to your
 [`extend-select`](https://docs.astral.sh/ruff/settings/#extend-select)._
 
+_Note: All of the above mentioned Notebook configurations will run the action for each cell individually.
+This is the way VS Code handles Notebook actions and is unrelated to `ruff-lsp`. If you'd prefer to run
+them on the entire notebook at once, prefer to use the `Ruff` prefixed commands such as
+`Ruff: Organize Imports` and `Ruff: Fix all auto-fixable problems`._
```

Comment:
This isn't true for "Format document" though, right? (At least, it wasn't in my testing.)

Reply:
It is true. Here, it sent 2 requests for a notebook containing 2 cells:
```
2023-11-08 08:20:15.130 [info] [Trace - 8:20:15 AM] Sending request 'textDocument/formatting - (15)'.
2023-11-08 08:20:15.130 [info] [Trace - 8:20:15 AM] Sending request 'textDocument/formatting - (16)'.
```

Even though VS Code runs them for each cell, they run it in parallel. This will seem like it performed the action on an entire Notebook.